src/RageUtil/Graphics/Display/RageDisplay.cpp

```diff
@@ -155,217 +179,13 @@ RageDisplay::SetVideoMode(VideoModeParams p, bool& bNeedReloadTextures)
 	return SETVIDEOMODE_FAILED.GetValue() + " " + join(";", vs);
 }
 
-void
-RageDisplay::ProcessStatsOnFlip()
-{
-	if (PREFSMAN->m_bShowStats || LOG_FPS) {
-		g_iFramesRenderedSinceLastCheck++;
-		g_iFramesRenderedSinceLastReset++;
-
-		const std::chrono::duration<double> timeDelta =
-		  std::chrono::steady_clock::now() - g_LastCheckTimer;
-		const auto checkTime = timeDelta.count();
-		if (checkTime >= 1.0) // update stats every 1 sec.
-		{
-			g_LastCheckTimer = std::chrono::steady_clock::now();
-			g_iNumChecksSinceLastReset++;
-			g_iFPS = static_cast<int>(
-			  g_iFramesRenderedSinceLastCheck / checkTime + 0.5);
-			g_iCFPS =
-			  g_iFramesRenderedSinceLastReset / g_iNumChecksSinceLastReset;
-			g_iCFPS = static_cast<int>(g_iCFPS / checkTime + 0.5);
-			g_iVPF =
-			  g_iVertsRenderedSinceLastCheck / g_iFramesRenderedSinceLastCheck;
-			g_iFramesRenderedSinceLastCheck = g_iVertsRenderedSinceLastCheck =
-			  0;
-			if (LOG_FPS && !(PREFSMAN->m_verbose_log > 1)) {
-				auto sStats = GetStats();
-				s_replace(sStats, "\n", ", ");
-				Locator::getLogger()->trace(sStats.c_str());
-			}
-		}
-	}
-}
-
-void
-RageDisplay::ResetStats()
-{
-	if (PREFSMAN->m_bShowStats || LOG_FPS) {
-		g_iFPS = g_iVPF = 0;
-		g_iFramesRenderedSinceLastCheck = g_iFramesRenderedSinceLastReset = 0;
-		g_iNumChecksSinceLastReset = 0;
-		g_iVertsRenderedSinceLastCheck = 0;
-		g_LastCheckTimer = std::chrono::steady_clock::now();
-	}
-}
-
-std::string
-RageDisplay::GetStats() const
-{
-	std::string s;
-	// If FPS == 0, we don't have stats yet.
-	if (!GetFPS())
-		s = "-- FPS\n-- av FPS\n-- VPF";
-
-	s = ssprintf("%i FPS\n%i av FPS\n%i VPF", GetFPS(), GetCumFPS(), GetVPF());
-
-	//	#ifdef _WIN32
-	s += "\n" + this->GetApiDescription();
```

Comment:
Since this PR keeps both implementations of display renderer functional, losing this line has removed useful debug information